lib/matplotlib/figure.py

```diff
@@ -1891,7 +1891,12 @@ def colorbar(self, mappable, cax=None, ax=None, use_gridspec=True, **kw):
             else:
                 cax, kw = cbar.make_axes(ax, **kw)
         cax._hold = True
-        cb = cbar.colorbar_factory(cax, mappable, **kw)
+
+        # need to remove kws that cannot be passed to Colorbar
+        NON_COLORBAR_KEYS = ['fraction', 'pad', 'shrink', 'aspect', 'anchor',
+                             'panchor']
```

Comment:
Does the remove of these kwargs mean the docstring was wrong/needs changing now?

Reply:
I don't think so the notes section in the docs [says](http://matplotlib.org/api/_as_gen/matplotlib.figure.Figure.html?highlight=figure%20colorbar#matplotlib.figure.Figure.colorbar) says:

Additional keyword arguments are of two kinds:
- axes properties
- colorbar properties

We receive a mixture, which AFAICS is intended. But we have to remove the kwargs, that `Colorbar` does not understand.